src/RoleObjects/TabListData.js

```diff
@@ -61,9 +63,10 @@ export default class TabListData extends CompositeData {
   }
 
   /**
-   * Keydown listener for when the tablist is pressed
-   * @access private
-   */
+  * Keydown listener for when the tablist is pressed
+  * @access private
+  */
+
```

Comment:
Nit: please remove the blank line. A function comment should be right up against its associated function.

Reply:
Done